src/player/index.ts

```diff
@@ -232,11 +302,36 @@ function getNextTour(): CodeTour | undefined {
   }
 }
 
-function getHeaderText(step: CodeTourStep): string {
+async function getFileContext(
+  line: number,
+  file: string,
+  uri: Uri
+): Promise<string> {
+  let label = `This step is on line ${line} in file ${file}`;
+  const document = await workspace.openTextDocument(uri);
+  if (document.languageId === "typescript") {
+    let tokens = await tokenize(document.getText().split("\n"));
```

Comment:
nit: unless I'm misreading, tokens can be a const instead of a let?

Reply:
Yeah, should be.